main.tf

```diff
@@ -1,28 +1,35 @@
 data "aws_caller_identity" "current" {}
 
-data "aws_sns_topic" "bucket-arn" {
-  name = "bucket-arn"
+resource "aws_s3_bucket" "bucket" {
+  bucket = aws_s3_bucket.bucket.id
 }
 
+resource "aws_s3_bucket_notification" "bucket_notification" {
```

Comment:
You will also want to make use of count, similar to the following line:
```274 count = var.replication_enabled ? 1 : 0```
Do not forget to define `var.notification_enabled` in your variables list as well.

Reply:
this is still not addressed